icons/Google/content.json

```diff
@@ -2,5 +2,5 @@
   "width": 15,
   "height": 15,
   "viewBox": "0 0 100 100",
-  "block": "<path d=\"M50.714 94.72c12.875-1.356 23.266-6.475 31.173-15.356 7.907-8.88 11.86-19.828 11.86-32.844 0-3.358-.209-5.855-.629-7.49l-42.194 4.447v15.745l25.82-2.722c-.42 1.863-1.12 3.861-2.1 5.994-.979 2.132-2.413 4.348-4.303 6.646-1.889 2.299-4.513 4.29-7.872 5.973-3.358 1.684-7.207 2.754-11.545 3.212-7.558.796-14.03-1.25-19.418-6.14s-8.082-11.22-8.082-18.986c0-7.767 2.694-14.699 8.082-20.795 5.388-6.096 11.86-9.542 19.418-10.339 7.417-.782 13.435.893 18.053 5.025l11.966-13.017c-7.978-7.137-18.054-10.063-30.23-8.78-12.455 1.313-23.02 6.766-31.698 16.357C10.338 31.242 6 42.265 6 54.72c0 12.456 4.373 22.561 13.12 30.316 8.747 7.755 19.278 10.983 31.594 9.685z\"/>"
+  "block": "<path d=\"M50.714 94.72c12.875-1.356 23.266-6.475 31.173-15.356 7.907-8.88 11.86-19.828 11.86-32.844 0-3.358-.209-5.855-.629-7.49l-42.194 4.447v15.745l25.82-2.722c-.42 1.863-1.12 3.861-2.1 5.994-.979 2.132-2.413 4.348-4.303 6.646-1.889 2.299-4.513 4.29-7.872 5.973-3.358 1.684-7.207 2.754-11.545 3.212-7.558.796-14.03-1.25-19.418-6.14s-8.082-11.22-8.082-18.986c0-7.767 2.694-14.699 8.082-20.795 5.388-6.096 11.86-9.542 19.418-10.339 7.417-.782 13.435.893 18.053 5.025l11.966-13.017c-7.978-7.137-18.054-10.063-30.23-8.78-12.455 1.313-23.02 6.766-31.698 16.357C10.338 31.242 6 42.265 6 54.72c0 12.456 4.373 22.561 13.12 30.316 8.747 7.755 19.278 10.983 31.594 9.685z\" fill=\"currentColor\"/>"
```

Comment:
No fill-rule for this one?

Reply:
Indeed, at first I wanted to apply the `fill-rule` but I saw that some icons use it and others don't.